ui/components/app/flask/snap-remove-warning/index.scss

```diff
@@ -0,0 +1,16 @@
+.snap-remove-warning {
+  color: black;
+
+  .popover-header {
```

Comment:
What are these styling changes for? Normally, we scope style changes to the namespace. In this case that would be `snap-remove-warning`. Making style changes like this can create dependencies between components which can be broken fairly easily. Ideally, we want the same styles across all popovers to maintain consistency. Can you post a screenshot of how the component looks without this rule set? I've been meaning to further customize the popover component so we can inject some custom css like you're having to need to do here.

Reply:
Sure, so this is a view without padding: 
![image](https://user-images.githubusercontent.com/38461171/156229012-662955c1-3fd1-4267-b6cf-ff79122b4b73.png)
And this is with current CSS: (notice the second phrase offset)
![image](https://user-images.githubusercontent.com/38461171/156229067-3f6d1305-1815-44d2-842f-4092b3af9a8b.png)